resources/web/wwi/Parser.js

```diff
@@ -1249,4 +1249,10 @@ function rotateHDR(image, rotate) {
   return rotatedbits;
 }
 
+function sanitizeHTML(text) {
```

Comment:
Shouldn't we make it private, as it is used only from within this class?
```suggestion
function _sanitizeHTML(text) {
```